controller/service.go

```diff
@@ -63,31 +69,56 @@ func (s *Service) CreateBirthday(ctx context.Context, req *pb.CreateBirthdayRequ
 	return convertedBirthday, nil
 }
 
+//GetBirthday is getting a birthday object
```

Comment:
`getting a birthday object` from what? by it's ID? private number? - it's not clear for an outside-observer. 
same for the other methods

Reply:
done